internal/cmd/root.go

```diff
@@ -19,6 +19,7 @@ var (
 	layersFlag  bool
 	outputFlag  enum
 	versionFlag bool
+	fileFlag    string
```

Comment:
Please sort alphabetically – and feel free to fix the sort error of the existing flags as well, since you're already on it 😄

Also, please rename to `filenameFlag` to make a little clearer what it's for, thx!